docs/using/backup.md

```diff
@@ -0,0 +1,93 @@
+# Backup and Restore
+
+> <b style="color:goldenrod">Caution</b> Backup and restore features are a technical preview and currently only work with MySQL database servers backing up to Amazon AWS S3 storage locations.
+
+## Before you start
```

Comment:
pmm-agent should be run as root in order to stop and restart MySQL using systemctl.

Reply:
`pmm-admin config` starts `pmm-agent` so if that command is sudo'd, won't pmm-agent also be root PID? But you raise a general point as to whether all commands should show they are run as `sudo` or should be assume the reader can work that out? In the case of docker containers, `sudo` fails because it's not in the provided image. I will defer this specific instance and do a global doc review of where/when/if to mention running all `pmm-admin` commands via sudo.

Reply:
As far as I know when we run `pmm-admin config` it actually performs a configuration (modifies configuration file pmm-agent.yaml) and this command requires pmm-agent to be already running via `pmm-agent run --server-address=1.2.3.4:443 --config-file=pmm-agent.yaml`. One more thing, it is required that the running agent should be on the same machine as MySQL e.g. the restore won't work if pmm-agent inside Docker container and MySQL is on the host machine.

Reply:
Thanks @oter for the clarifications. These are general pointers that we will cover in other sections. I have a place in mind and will tackle it separately. If you think it's important to give these tips on this page (right next to the commands), let me know.